src/main/scala/org/hammerlab/guacamole/assembly/DeBruijnGraph.scala

```diff
@@ -123,13 +125,13 @@ class DeBruijnGraph(val kmerSize: Int,
   }
 
   /**
-   *  Searches forward or backward from a node to find those connected by a unique path
+    *  Searches forward or backward from a node to find those connected by a unique path
```

Comment:
I ran some scripts recently and like 90% of the codebase aligns these the other way, and I think that's more standard in the world at large, so I'd vote we do that! I was going to send a quick PR doing it everywhere but maybe undo this for now?

Make sure you just check the box in intellij and then you'll never have to think about it again.